src/store/reducers/index.ts

```diff
@@ -13,6 +14,7 @@ export const VenueTemplateReducers: { [key: string]: Reducer } = {};
 export const MiscReducers = {
   room: roomReducer,
   chat: chatReducer,
+  userProfile: userProfileReducer,
   location: locationReducer,
   attendance: attendanceReducer,
   sovereignVenue: sovereignVenueReducer,
```

Comment:
While it's not true currently, it would be good to keep these in alphabetic order.